dbms/src/main/java/org/polypheny/db/partition/HashPartitionManager.java

```diff
@@ -35,34 +36,36 @@ public class HashPartitionManager extends AbstractPartitionManager {
 
 
     @Override
-    public long getTargetPartitionId( CatalogTable catalogTable, String columnValue ) {
+    public long getTargetPartitionId( LogicalTable table, PartitionProperty property, String columnValue ) {
         long hashValue = columnValue.hashCode() * -1;
 
         // Don't want any neg. value for now
         if ( hashValue <= 0 ) {
             hashValue *= -1;
         }
```

Comment:
This is unnecessarily complex. The probability for a hash to be negative should be roughly similar to the probability of it being positive. Thus, the first *-1 is unnecessary. However, even simpler would be to do a Math.abs(columnValue.hashCode()).

Reply:
Agreed, adjusted.